### DISCUSSION TITLE
[Feature] Album with selected people

### DISCUSSION THREAD

**Discussion by maxim31cote** [immich-app/immich#13378] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Have the ability to create smart albums that would automatically add chosen people recognized by Facial Recognition, with the ability to share them

Example: My kids Album, My parents Album etc.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This will be covered by #1673
